### PR TITLE
all crud operations completed(get,put delete)

### PR DESCRIPTION
**Pull request by VinayV9**
updating a note 
deleting a note
getting  specified note
done

### CODE REVIEW

File: server/controllers/notes.js [hunk 1]
```diff
@@ -52,8 +79,18 @@ const notesController = {
...
+        Note.find({sulg : sulgId})
+            .then(notes => {
+                notes = notes || []
+                return res.status(200).json({
+                    error: false,
+                    notes: notes, 
+                }) 
+            })
+            .catch(error => {
+
```
Comment (skyshader) [inline server/controllers/notes.js]: Lets return a 404?

**Review by skyshader — COMMENTED** [review]
The PR seems good! Just check the 404 thing in get API method.

**Comment by whimsicaldreamer** [comment]
Check the variable `slugID` spelling on line 83.

File: server/controllers/notes.js [hunk 2]
```diff
@@ -24,13 +24,40 @@ const notesController = {
...
     delete: (req, res) => {
-        console.log(`delete: ${req.params}`)
-        res.send('ok')
+        let slugId = req.params.id
+
+        Note.update({slug:slugId}, {$set: { deleted: true} })
+            .then(note =>{
+                return res.status(200).json({
+                    error: false,
+                    message: 'Note delted',
```
Comment (whimsicaldreamer) [inline server/controllers/notes.js]: Missed an `e` 😅

Comment (VinayV9) [reply server/controllers/notes.js]: yeah

**Comment by skyshader** [comment]
Closing it as we received another PR without much of back and forth. Please do submit another PR and be active on it. Thanks for your contribution :)

**Pull request closed without merging**